lib/core-integration/src/Test/Integration/Scenario/API/Wallets.hs

```diff
@@ -177,6 +177,37 @@ spec = do
             , expectFieldNotEqual passphraseLastUpdate Nothing
             ]
 
+    it "CREATE_WALLET_01 - SQL injection attempt when creating a wallet" $ \ctx -> do
+        let mnemonics = [ "pulp", "ten", "light", "rhythm", "replace"
+                        , "vessel", "slow", "drift", "kingdom", "amazing"
+                        , "negative", "join", "auction", "ugly", "symptom"] :: [Text]
+        let payload = Json [json| {
+                "name": "new wallet\",'',''); DROP TABLE \"wallet\"; --",
```

Comment:
why `"new wallet\",'','');` it follows from https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/src/Cardano/Wallet/DB/Sqlite.hs#L585 (so after name there are two more fields to be filled) and the use of `insert_` https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/src/Cardano/Wallet/DB/Sqlite.hs#L415

Reply:
interesting fact : When I replaced "insert_" with wrongly constructing of SQL text by hand (using T.intersperse) and then rawExecute) I was managed to produce SQL injection, and drop wallet table .... So escaping used in persistent-sqlite and persistent (https://github.com/yesodweb/persistent/blob/53fa7589db2cabe586d506fea12006b759b1ce79/persistent-sqlite/Database/Persist/Sqlite.hs#L483) works properly for this kind of injections (so called `First Order SQL injection attacks`)